webapp/src/pages/Users/Users.tsx

```diff
@@ -51,12 +51,12 @@ export default function Users() {
 
   useEffect(() => {
     const fetchUsersList = async () => {
-      const lUsersList = await getRows(`${USERS_EP_URL}`);
+      const lUsersList = await getRows(`${USERS_EP_URL}?search=${search}`);
       setUsersList({ ...lUsersList });
       return lUsersList;
     };
     fetchUsersList().catch((e) => console.error(e));
-  }, []);
+  }, [search]);
```

Comment:
praise: Good job for making this work with the backend :+1:

Reply:
Thanks